.github/workflows/unit_test_200gb_CI.yml

```diff
@@ -76,12 +76,42 @@ jobs:
 
       - name: Start Vespa
         run: |
+          retry_command() {
+            local n=1
+            local max=5
+            local delay=5
+            while true; do
+                "$@" && break || {
+                    if [[ $n -lt $max ]]; then
+                        ((n++))
+                        echo "Command failed. Attempt $n/$max:"
+                        sleep $delay;
+                    else
+                        echo "The command has failed after $n attempts."
+                        return 1
+                    fi
+                }
+            done
+          }
+          
+          echo "Updating package list"
+          retry_command apt-get update -y
+          
+          # Build Vespa components
+          echo "Installing jdk 17"
```

Comment:
Consider using github action plugin for this: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven#caching-dependencies 
Here are some java distributions: https://github.com/actions/setup-java?tab=readme-ov-file#supported-distributions, we can start with temurin or corretto, version 17

Reply:
noted it for fast follow, might not be able to implement it in this iteration due to urgency